src/main/java/seedu/address/model/task/RecurType.java

```diff
@@ -4,16 +4,28 @@
     DAILY {
         @Override
         public long getInterval() {
-            // return 1000l*60*60*24;
-            return 15000l; // for testing
+            return 1000l * 60 * 60 * 24;
```

Comment:
Might there be a better way to express these "magic numbers"? perhaps, as final integers.

Reply:
Fixed